notebooks/tabular_ML_regression_non_tree_based.ipynb

```diff
@@ -0,0 +1,263 @@
+    "vscode": {
+     "languageId": "python"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "X_train, X_test, y_train, y_test, df_train, df_test = create_and_split_dataframe() "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Train model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "python"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "svr = SVR(kernel='rbf', gamma=0.1)\n",
+    "svr.fit(X_train, y_train)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "python"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "y_pred = svr.predict(X_test) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "python"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "mae_svr = mean_absolute_error(y_test, y_pred)\n",
+    "mse_svr = mean_squared_error(y_test, y_pred)\n",
+    "r2_svr = r2_score(y_test, y_pred)\n",
+    "\n",
+    "print(\"MAE : \", mae_svr)\n",
+    "print(\"MSE : \", mse_svr)\n",
+    "print(\"R2 : \", r2_svr)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Make intelligibility with readml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**WARNINGS**: Take care of change de config_local.cfg to adapt it to the use case, you may need to re run ```pip install -e .``` after the change in the configuration."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "python"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "model_explain = svr # The model you use\n",
+    "task = \"regression\" # here we try to solve a regression problem\n",
+    "tree_based_model = False # if you use a tree based model\n",
+    "features_name = list(X_train.columns) # all the features without de target column\n",
+    "features_to_interpret = features_name # the features on which you want to make intelligibility\n",
+    "target_col = list(y_train.columns)[0] # the tagets columns\n",
+    "out_path = \"../outputs/notebooks/\" # the path where you want to save the report\n",
+    "res_dir = os.path.join(ROOT_DIR, out_path)\n",
+    "if not os.path.exists(res_dir)\n",
+    "    os.makedirs(res_dir)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "python"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "exp = ExplainML(\n",
+    "        model=model_explain,\n",
+    "        task_name=task,\n",
+    "        tree_based_model=tree_based_model,\n",
+    "        features_name=features_name,\n",
+    "        features_to_interpret=features_to_interpret,\n",
+    "        target_col=target_col,\n",
+    "        out_path=out_path,\n",
```

Comment:
I guess you should use :

out_path=res_dir

instead of :

out_path=out_path